pyvista/plotting/camera.py

```diff
@@ -276,3 +274,31 @@ def __del__(self):
         """Delete the camera."""
         self.RemoveAllObservers()
         self.parent = None
+
+    @property
+    def view_angle(self):
+        """Return the camera view angle.
+
+        Examples
+        --------
+        >>> import pyvista
+        >>> plotter = pyvista.Plotter()
+        >>> plotter.camera.view_angle
+        30.0
+
+        """
+        return self.GetViewAngle()
+
+    @property
+    def direction(self):
+        """Get the vector in the direction from the camera position to the focal point.
```

Comment:
```suggestion
        """The vector in the direction from the camera position to the focal point.
```

NITPICK: Properties are nouns.

Reply:
Thanks!